docs/deployment/configuration/auth_setup.rst

```diff
@@ -351,28 +351,27 @@ Apply OIDC Configuration
            # Use the non-encoded version of the random password 
              clientSecret: "<your-random-password>" 
              clientId: flytepropeller
-         
-      8. Save and exit your editor.
 
-      9. Restart ``flyteadmin`` for the changes to take effect:
+.. note::
 
-      .. prompt:: bash $
+   For `multi-cluster deployments <https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html>`__, this secret definition has to be added to the `values-dataplane.yaml` file.
+   There's no need to create this secret in the control plane cluster if you're not running `flytepropeller` there.
 
-         kubectl rollout restart deployment/flyteadmin -n flyte
+      8. Save and exit your editor.
 
-      10. Restart ``flytepropeller`` to start using authenticated requests:
+      9. Upgrade your Helm release with the new configuration:
 
       .. prompt:: bash $
 
-          kubectl rollout restart deployment/flytepropeller -n flyte
+         helm upgrade <release-name> flyteorg/flyte-binary -n <your-namespace> --values <your-values-file>.yaml
 
-      11. Restart ``flytescheduler``` to start using authenticated requests:
+      10. Verify that the `flytepropeller`, `flytescheduler` and `flyteadmin` Pods are restarted and Running: 
```

Comment:
What's the reason to remove the instructions on how to restart the deployments (`kubectl rollout restart ...`) in the docs?